ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -248,6 +248,21 @@ objects:
           --zone "${GOOGLE_COMPUTE_ZONE}" \
           packer@"${INSTANCE_PREFIX}" \
           --command 'export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${RELEASE_IMAGE_LATEST}" && timeout 4800 bash -ce "create-cluster nested 2>&1 | tee output"' | tee /tmp/artifacts/output-e2e-libvirt
+        echo 'Collect all the info about clusteroperators'
+        LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
+          --zone "${GOOGLE_COMPUTE_ZONE}" \
+          packer@"${INSTANCE_PREFIX}" \
+          --command 'export KUBECONFIG=/home/$USER/clusters/nested/auth/kubeconfig && bash -ce "oc get co | tee output"' | tee /tmp/artifacts/output-e2e-libvirt
+        echo 'Extract openshift-must-gather from the image and put it to executable path'
+        LD_PRELOAD=/usr/lib64/libnss_wrapper.so gcloud compute --project "${GOOGLE_PROJECT_ID}" ssh \
+          --zone "${GOOGLE_COMPUTE_ZONE}" \
+          packer@"${INSTANCE_PREFIX}" \
+          --command 'mkdir -p /tmp/must-gather && oc image extract $(KUBECONFIG=/home/$USER/clusters/nested/auth/kubeconfig oc adm release info --image-for must-gather) --path /usr/bin/openshift-must-gather:/tmp/must-gather && chmod +x /tmp/must-gather/openshift-must-gather'
```

Comment:
No need for the kubeconfig is there, since you know the release image without having to ask the cluster.  Can we use:

```sh
oc image extract $(oc adm release info --image-for must-gather "${RELEASE_IMAGE_LATEST}") --path /usr/bin/openshift-must-gather:/tmp/must-gather
```